kubemarine/resources/configurations/defaults.yaml

```diff
@@ -63,6 +63,9 @@ services:
         PollIntervalMinSec: 32
         PollIntervalMaxSec: 2048
 
+  kernel_linux:
+    version: 5.4.0-135-generic
```

Comment:
Do we expect that the parameter can be redefined by the user?

Reply:
Maybe this parameter will be changed in the future

Reply:
rework procedure